app/javascript/shared/helpers/BaseActionCableConnector.js

```diff
@@ -28,17 +32,35 @@ class BaseActionCableConnector {
 
     setInterval(() => {
```

Comment:
Another unrelated comment, can we see if we can replace this with `setTimeout`?  Generally, `setInterval` is not considered safe unless we know when to trigger a `clearInterval`. For `setInterval`, the callback function stays in memory until `clearInterval` is called. This can cause potential memory issues. 

We could make a function `triggerPresenceInterval.`

```js
triggerPresenceInterval() {
    setTimeout(() => {
      this.subscription.updatePresence();
      const isConnectionActive = this.consumer.connection.isOpen();
      const shouldRefetch =
        BaseActionCableConnector.isDisconnected && isConnectionActive;
      if (shouldRefetch) {
        this.refreshActiveConversationMessages();
        BaseActionCableConnector.isDisconnected = false;
      }
      this.triggerPresenceInterval();
    }, PRESENCE_INTERVAL);
  }
```
and then call `this.triggerPresenceInterval()` in the constructor